ouster-ros/config/driver_params.yaml

```diff
@@ -0,0 +1,25 @@
+ouster/os_driver:
```

Comment:
possibly im not looking in the right place. 

think it would help to specify what the behavior is if you don't fill these out / leave as is, so for ex sensor_hostname is necessary but the rest seem to be optional

Reply:
makes sense. I will throw few comments.